keras/preprocessing/image.py

```diff
@@ -364,6 +418,9 @@ class ImageDataGenerator(object):
         rescale: rescaling factor. If None or 0, no rescaling is applied,
             otherwise we multiply the data by the value provided
             (before applying any other transformation).
+        hsv_augmentation: parameters for HSV data augmentation
+                          (hue shift, saturation scale, saturation shift,
+                          value scale, and value shift)
```

Comment:
+1 on "sensible default values" should be in the docs or in the definition itself.